eventstore/jpa/src/main/java/org/occurrent/eventstore/jpa/JPAEventLog.java

```diff
@@ -0,0 +1,9 @@
+package org.occurrent.eventstore.jpa;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.JpaSpecificationExecutor;
+import org.springframework.data.repository.NoRepositoryBean;
+
+@NoRepositoryBean
+public interface JPAEventLog<T extends CloudEventDaoTraits>
+    extends JpaRepository<T, Long>, JpaSpecificationExecutor<T> {}
```

Comment:
```suggestion
@NoRepositoryBean
public interface JPAEventLog<T extends CloudEventDaoTraits, TKey>
    extends JpaRepository<T, TKey>, JpaSpecificationExecutor<T> {}
```

Had `Long` key in as a placeholder